### PR TITLE
Add subsections in "For Users"

### PR DESCRIPTION
**Pull request by pireddag**
This time t looks more sensible. I do not know why the environment 
<\references>
  <\collection>
I have no control on that.

### CODE REVIEW

File: src/awesome-texmacs.tm [hunk 1]
```diff
@@ -38,12 +38,22 @@
 
   <section*|For Users>
 
+  <subsection|Documents written with TeXmacs>
+
```
Comment (pireddag) [inline src/awesome-texmacs.tm:42]: This blank line was unintended. Blank lines are more difficult to control in TeXmacs than in TeX

File: src/awesome-texmacs.tm [hunk 2]
```diff
@@ -69,11 +79,14 @@
...
-    <associate|auto-4|<tuple|<with|mode|<quote|math>|\<bullet\>>|?|../../../.TeXmacs/texts/scratch/no_name_2.tm>>
-    <associate|auto-5|<tuple|<with|mode|<quote|math>|\<bullet\>>|?|../../../.TeXmacs/texts/scratch/no_name_2.tm>>
+    <associate|auto-1|<tuple|?|?>>
+    <associate|auto-2|<tuple|?|?>>
+    <associate|auto-3|<tuple|<with|mode|<quote|math>|\<bullet\>>|?>>
+    <associate|auto-4|<tuple|<with|mode|<quote|math>|\<bullet\>>|?>>
+    <associate|auto-5|<tuple|1|?>>
+    <associate|auto-6|<tuple|2|?>>
+    <associate|auto-7|<tuple|3|?>>
+    <associate|auto-8|<tuple|<with|mode|<quote|math>|\<bullet\>>|?>>
```
Comment (pireddag) [inline src/awesome-texmacs.tm:89]: I do not know why this change happened

File: src/awesome-texmacs.tm (hunk 1 above)
Comment (mgubi) [reply src/awesome-texmacs.tm:42]: I think is ok.

File: src/awesome-texmacs.tm (hunk 2 above)
Comment (mgubi) [reply src/awesome-texmacs.tm:89]: Yes, this looks strange. But I think we can just let them stay.

**Review by mgubi — APPROVED** [review]
Ok. it looks fine. I will try to merge. (my first time)

**Comment by pireddag** [comment]
Will write then later my part of the procedure (fork-clone-add-commit-push-request pull). I have though to figure out how to keep my fork updated.

**Comment by mgubi** [comment]
There is no "fork" as far as I understand. There is "clone a repository" and "create a new branch in a repository". What you do is create branches for making pull requests. When they are approved you can just pull them from the remote repository in your local repository and they will appear in the main branch. At this point you can delete your local PR branch.

**Comment by pireddag** [comment]
Could you please make the opposite test, i.e. try to make a pull request on one of my repositories that you did not fork (that is, any of them except formatRef)? I do not see the way to create a branch in the texmacs/notes repository.
I have written the sequence that I followed in the readme and submitted it as a pull request, but if what I wrote is wrong (or not the best way) I will rewrite it.
On-topic I noticed that if I fork I can keep the fork updated with the git commands fetch upstream and merge upstream/main, which seems better than cloning every time.

**Comment by pireddag** [comment]
https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request

> Create a pull request to propose and collaborate on changes to a repository. These changes are proposed in a branch, which ensures that the default branch only contains finished and approved work.
> Anyone with read permissions to a repository can create a pull request, but you must have write permissions to create a branch. If you want to create a new branch for your pull request and don't have write permissions to the repository, you can fork the repository first. For more information, see "Creating a pull request from a fork" and "About forks."

**Comment by mgubi** [comment]
Dear Giuseppe. I will look. Anyway I do not understand why you use the word "fork". You have to clone the repository only once. Then on your clone you can have your own branches and also branch from "origin/main" in order to prepare pull requests.

**Comment by mgubi** [comment]
you do not have to create a branch on texmacs/notes, you create a branch on your clone of texmacs/notes and then push the branch upstream to 'origin' to start the PR. If you clone locally you own GitHub clone of texmacs/notes then this will not work.

**Comment by pireddag** [comment]
I even haven't figured out how to clone the repository "not locally", i.e. on the GitHub servers. I am only able to clone it locally.

**Comment by mgubi** [comment]
Sorry. I see now your point. The instructions are correct. You have to 'fork' (i.e. clone within GitHub) and then push your changes into your fork, form that you can initiate a PR for a branch in your fork into a branch in the main repository.

**Comment by pireddag** [comment]
In fact now I tried to type "git push" from a clone of the texmacs/notes directory and I obtained
remote: Permission to texmacs/notes.git denied to pireddag.
fatal: unable to access 'https://github.com/texmacs/notes.git/': The requested URL returned error: 403
